internal/grpc/handler.go

```diff
@@ -113,6 +124,23 @@ func getNetworkAddress(address string) (string, string, error) {
 	}
 }
 
+func (h *handler) getClientTransportCredentials() (credentials.TransportCredentials, error) {
+	var creds credentials.TransportCredentials
```

Comment:
This should be:

```
if !h.tls {
  return nil, nl
}
```

To match the rest of this repo, this no need to declare creds as a var

Reply:
got it. ty

Reply:
So if I do that I still need to declare the var:
`
        if !h.tls {
		return nil, nil
	}
	var err error
	var creds credentials.TransportCredentials
	creds, err = grpcurl.ClientTransportCredentials(h.insecure, h.cacert, h.cert, h.key)
	if err != nil {
		return nil, err
	}
	if h.serverName != "" {
		if err := creds.OverrideServerName(h.serverName); err != nil {
			return nil, err
		}
	}
	return creds, nil
`